src/routes/(console)/project-[region]-[project]/databases/database-[database]/table-[table]/export/+page.svelte

```diff
@@ -0,0 +1,327 @@
+                                <Icon size="s" icon={IconInfo} />
+                                <span slot="tooltip">
+                                    Define how to separate values in the exported file.
+                                </span>
+                            </Tooltip>
+                        </Layout.Stack>
+                    </InputSelect>
+
+                    <InputCheckbox
+                        id="includeHeader"
+                        label="Include header row"
+                        description="Column names will be added as the first row in the CSV"
+                        bind:checked={includeHeader} />
+
+                    <Layout.Stack gap="m">
+                        <div class:disabled-checkbox={localTags.length === 0}>
+                            <InputCheckbox
+                                id="exportWithFilters"
+                                label="Export with filters"
+                                description="Export rows that match the current table filters"
+                                disabled={localTags.length === 0}
+                                bind:checked={exportWithFilters} />
+                        </div>
+
+                        {#if localTags.length > 0}
+                            <ul
+                                class="u-flex u-flex-wrap u-cross-center u-gap-8 tags"
+                                style="padding-left: 1.75rem;">
+                                <TagList
+                                    tags={localTags}
+                                    on:remove={(e) => {
+                                        removeLocalFilter(e.detail);
+                                    }} />
+                            </ul>
+                        {/if}
+                    </Layout.Stack>
+
+                    <InputCheckbox
+                        id="emailOnComplete"
+                        label="Email me when export is complete"
+                        description="Useful for large exports that may take a few minutes"
+                        bind:checked={emailOnComplete} />
+                </Layout.Stack>
+            </Fieldset>
+        </Layout.Stack>
+    </Form>
+    <svelte:fragment slot="footer">
+        <Layout.Stack justifyContent="flex-end" direction="row">
+            <Button fullWidthMobile secondary on:click={() => (showExitModal = true)}>
+                Cancel
+            </Button>
+            <Button
+                fullWidthMobile
+                on:click={() => formComponent.triggerSubmit()}
+                disabled={$isSubmitting || !selectedBucket || selectedColumnCount === 0}>
+                Export
+            </Button>
+        </Layout.Stack>
+    </svelte:fragment>
+</Wizard>
+
+<style>
+    .column-grid {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+        row-gap: 1.25rem;
+        column-gap: 1rem;
+    }
+
+    .show-more-wrapper {
+        margin-bottom: -0.5rem;
```

Comment:
inline the style if only used once like - `<div style:margin-bottom="0.5rem">...</div>`